javascript/builtins/Intl.json

```diff
@@ -1082,6 +1089,70 @@
               }
             }
           },
+          "formatToParts": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat/formatToParts",
+              "support": {
+                "webview_android": {
+                  "version_added": false
+                },
+                "chrome": {
+                  "version_added": "63",
+                  "flag": {
+                    "type": "preference",
```

Comment:
Should be `compile_flag`, I guess, if it is not a real preference but controllable using the commandline.

Reply:
Yes, it's controlled by a command line flag, but it's not a compile time flag, either. If I have to choose between the two, it's closer to 'preferecne' because it can be controlled at run-time. 

The best solution is to add a 3rd enum for flag type: 'command line flag'.

Reply:
I agree to add a 3rd enum and did so in PR #615 .